integration-tests/tests/run-analysis.js

```diff
@@ -30,12 +30,12 @@ const testRunAnalysisFn = _.flowRight(
   await gotoAnalysisTab(page, token, testUrl, workspaceName);
 
   // Create analysis file
-  await click(page, clickable({ textContains: 'Start' }));
+  await click(page, clickable({ textContains: 'Start', isEnabled: true }));
```

Comment:
`isEnabled` defaults to true already.

https://github.com/DataBiosphere/terra-ui/blob/ba6d6779bef8574cf51166191e9192b1f9b42cf8/integration-tests/utils/integration-utils.js#L73-L78

Could the issue be that `clickable` only looks at `aria-disabled` (not `disabled`)?

Reply:
Could be! I may also have put a spinner overlay on something with an enabled buton, perhaps.

Reply:
Fixed this - just needed to await a spinner in a place previously untouched by spinners.